src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -384,6 +395,124 @@ type Currency {
   digits: Int!
 }
 
+"Stoptime represents the time when a specific trip on a specific date arrives to and/or departs from a specific stop."
+type DatedStopTime {
+  "Scheduled arrival time to the stop and a realtime estimate, if one exists."
+  arrival: ArrivalDepartureTime
+  "Scheduled departure time from the stop and a realtime estimate, if one exists."
+  departure: ArrivalDepartureTime
+  """
+  Whether the vehicle can be disembarked at this stop. This field can also be
+  used to indicate if disembarkation is possible only with special arrangements.
+  """
+  dropoffType: PickupDropoffType
+  """
+  Vehicle headsign of the trip on this stop. Trip headsigns can change during
+  the trip (e.g. on routes which run on loops), so this value should be used
+  instead of `tripHeadsign` to display the headsign relevant to the user.
+  """
+  headsign(
+    """
+    If translated headsign is found from gtfs translation.txt and wanted language is not same as
+    feed's language then returns wanted translation. Otherwise uses name from trip_headsign.txt.
+    """
+    language: String
+  ): String
+  """
+  Whether the vehicle can be boarded at this stop. This field can also be used
+  to indicate if boarding is possible only with special arrangements.
+  """
+  pickupType: PickupDropoffType
+  "Whether stop has been updated through a realtime update and if so, how."
+  realtimeState: StopRealTimeState!
```

Comment:
How to take planned cancellation into count here? Should we rename this field so it's not only realtime stuff or have a separate field?